perf/runner/src/index.ts

```diff
@@ -37,7 +37,7 @@ async function main(clientPublicIP: string, serverPublicIP: string) {
                  uploadBytes: 1,
                  downloadBytes: 1,
                  unit: "s",
-                 iterations: 100,
+                 iterations: iterations * 10,
```

Comment:
I don't think this is intuitive. When I set `--iterations 10` I don't expect `100` iterations to run instead.